009-strimzi-ui.md

```diff
@@ -147,11 +147,20 @@ spec:
     uiConfig: <mounted config map> # (3)
     backend:
     - name: 'strimzi-api' # (4)
-      address: 'https://route-to-strimzi-api-for-dev-cluster:port' # (5)
-      version: 1 # (6)
-      tls: # 7
+      type: [admin/...] # specific type to identify admin server?  # (5)
+      address: 'https://route-to-strimzi-api-for-dev-cluster:port' # (6)
+      version: 1 # (7)
+      tls: # (8)
         cert: <mounted tls certificate>
         version: TLS_1.3
+      authentication: (9)
+        type: [oauth/basic/...]
+        tokenServer: # for oauth
```

Comment:
Could we have these as 9.A / 9.B etc? I had comments like this originally, and Paolo asked (for reasability) to move them out. Also gives us the option to add to them and not effect the pseudo CR